### PR TITLE
util: remove unused backup until functions

### PR DESCRIPTION
**Pull request by fanminshi**
ref: #1626

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

File: pkg/util/k8sutil/backup.go [hunk 1]
```diff
@@ -14,244 +14,5 @@
...
-	if err != nil {
-		wErr := fmt.Errorf("fail to wait PVC (%s) '(%v)/Bound': %v", name, claim.Status.Phase, err)
-		return wErr
-	}
-
-	return nil
-}
-
+// What's this?
 var BackupImage = "quay.io/coreos/etcd-operator:latest"
```
Comment (hasbro17) [inline pkg/util/k8sutil/backup.go]: I think this was the image used for the backup sidecar pod. So we probably don't need this as well.

Comment (hongchaodeng) [reply pkg/util/k8sutil/backup.go]: Yep. Just delete it.

Comment (fanminshi) [reply pkg/util/k8sutil/backup.go]: @hasbro17 it is used here https://github.com/coreos/etcd-operator/blob/master/hack/build/operator/build#L27

Comment (hongchaodeng) [reply pkg/util/k8sutil/backup.go]: That's the build flag used for backup sidecar previously.
Delete that as well

Comment (fanminshi) [reply pkg/util/k8sutil/backup.go]: done. PTAL @hongchaodeng @hasbro17

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**